masterlist.yaml

```
@@ -3221,6 +3221,33 @@ plugins:
       - Hair
       - R.Relations
 
+  - name: '(Kart Hair.*|karthair.*)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/48588/'
+        name: 'Lore-Friendly Replacer for Default Hair for OCO v2'
+  - name: '(Kart Hair Replace v01|Kart Hair A Less Rude Mohawk)\.esp'
```

The regexes can be simplified a bit, especially considering the fact that LOOT evaluates regexes with case-insensitivity:

```suggestion
  - name: 'Kart ?Hair.*\.esp'
    url:
      - link: 'https://www.nexusmods.com/oblivion/mods/48588/'
        name: 'Lore-Friendly Replacer for Default Hair for OCO v2'
  - name: 'Kart Hair (Replace v01|A Less Rude Mohawk)\.esp'
```